addons/hr_timesheet/views/hr_timesheet_portal_templates.xml

```diff
@@ -86,6 +88,9 @@
                                     <t t-else="">
                                         Total: <span t-esc="hours_spent" t-options='{"widget": "float_time"}'/>
                                     </t>
+                                    <t t-if="should_display_allocated_hours and allocated_hours > 0">
```

Comment:
you can move the condition to your python logic, if we need the same attribute in another place, we will avoid duplicating the code, and if one day, we want to change the condition we fix it in one place